internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -102,6 +105,9 @@ func (*Sequencer) Initialize(r runtime.Runtime) []runtime.Phase {
 		).Append(
 			"udevd",
 			StartUdevd,
+		).Append(
```

Comment:
we can merge the two phases (udevd & machined) into a single one, so that they start in parallel and we have better startup time ;)

Reply:
yep, done